### ISSUE TITLE
Minor: typo in settings. "Poniter" should be "Pointer"

### ISSUE THREAD

**Issue by raptormama** [BelegCufea/CompassHUD#7]
In pointer types' Textures tab, "Pointer" is misspelled as "Poniter". This is cosmetic only and doesn't affect functionality.

**Comment by BelegCufea** [comment]
Thanks @raptormama 
I am bad at spelling (dysgraphia) in my native language, not to mention in any other language I don't know (like English). There are many mistakes, I'm sure! I will fix this one. If you find any others, I would be grateful if you reported them as well.

**Comment by raptormama** [comment]
Of course! I'm happy to if I see anything else.

Your English is excellent, and your addon is absolutely lovely. I wouldn't have guessed that you were not a native speaker.

**Comment by raptormama** [comment]
In the main Settings tab, at least in the Heading branch, "adjustment" appears at one point as "adujustment". Below that, "horizontally" appears as "horizontaly". (A great button to have! I hit it after every position tweak).

**Comment by BelegCufea** [comment]
Good catch, thank you very much. Will be fixed in the next release. Keep them comming :-)
